packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -100,6 +100,12 @@ export class CdkToolkit {
       }
     }
 
+    const outputsPath = options.outputsPath;
+    if (outputsPath) {
+      fs.ensureFileSync(outputsPath);
+      fs.writeJSONSync(outputsPath, {}, {encoding: 'utf8'});
```

Comment:
Please change to async:

```ts
await fs.ensureFile(outputsPath);
await fs.writeJson(outputsPath, ...)
```

Reply:
I think the `ensureFile` still needs to be blocking before trying to write to it no?